dll/cpl/desk/lang/ro-RO.rc

```diff
@@ -128,7 +129,7 @@ BEGIN
             "button", BS_AUTOCHECKBOX | WS_TABSTOP, 10, 110, 267, 19
     CONTROL "&Amână sublinierea literelor navigării din tastatură până la apăsarea tastei «Alt»",
             IDC_EFFAPPEARANCE_KEYBOARDCUES, "button", BS_AUTOCHECKBOX | WS_TABSTOP, 10, 125, 267, 19
-    CONTROL "Use &flat menus", IDC_EFFAPPEARANCE_FLATMENUS,
+    CONTROL "Folosește meniuri &plate", IDC_EFFAPPEARANCE_FLATMENUS,
```

Comment:
There is a problem with access key assignation to letter „p” here.
First, it's because for this form it was already assigned on the control with label „&Pictograme mari”, therefore for it to properly work you should assign other letter. Just for the record, in this form the access keys assigned so far are „a”, „c”, „e”, „f”, „m”, „r”, „p”, and „u”. One of the unassigned letters is, for example, „o”.
Second, regarding assigning access keys in general, Microsoft advises us to „avoid using letters that are difficult to see when they are assigned to access keys” then enlists „letters with descenders, such as p, g, q, or y” (see [Guidelines for Keyboard User Interface Design](https://learn.microsoft.com/ro-ro/previous-versions/windows/desktop/dnacc/guidelines-for-keyboard-user-interface-design?redirectedfrom=MSDN#atg_keyboardshortcuts_selecting_access_keys), which has also been included at the 10th point in [Romanian translation notes.txt](https://github.com/reactos/reactos/blob/master/media/doc/Romanian%20translation%20notes.txt)).